ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+* What is the current behavior?
+
+
+* If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem via
+https://plnkr.co or similar (you can use this template as a starting point: http://plnkr.co/edit/tpl:yBpEi4):
```

Comment:
Add commented script tags for different versions in the template (including snapshot)